src/test/java/org/folio/rest/api/ItemShelvingOrderMigrationServiceApiTest.java

```diff
@@ -38,7 +38,7 @@ public void shouldPopulateShelvingOrder() throws Exception {
 
   @Test
   public void shouldStopProcessingWhenCannotConvertItem() throws Exception {
-    var item = createItem(0);
+    var item = createItem(300);
```

Comment:
Rather than pass in an index, could this be changed to make it take the whole barcode. I think that would make the intent more explicit.

Reply:
The passed index is not only used for the barcode but also for the call number components.

Reply:
Ok, those could be split, however it's likely out of scope of this work